sdk/src/vm.rs

```diff
@@ -9,14 +10,31 @@ pub const NO_DATA_BLOCK_ID: u32 = 0;
 
 /// Abort execution.
 pub fn abort(code: u32, message: Option<&str>) -> ! {
+    exit(code, message, None)
+}
+
+/// Exit from current message execution, with the specified code and an optional message and data.
+pub fn exit(code: u32, message: Option<&str>, data: Option<RawBytes>) -> ! {
```

Comment:
We can probably just pass `RawBytes`, no need for the option here.

We always use an empty "raw bytes" to indicate "nothing" (because that's not valid cbor anyways).

Reply:
E.g., see `send`.

(all this will change with #987 anyways)

Reply:
Ok, fair enough, that's a rust thing :)

Reply:
Oh, no, this isn't a rust thing. This is just a legacy FVM thing that needs to be fixed.